.github/workflows/publish-runtime.yml

```diff
@@ -0,0 +1,166 @@
+name: Publish Runtime Draft
+
+# The code (like generate-release-body) will be taken from the tag versions, not master
+on:
+  workflow_dispatch:
+    inputs:
+      from:
+        description: tag (ex. runtime-53) to retrieve commit diff from
+        required: true
+      to:
+        description: tag (ex. runtime-155) to generate release note and srtool runtimes from
+        required: true
+
+jobs:
+  ####### Build runtimes with srtool #######
+
+  setup-scripts:
+    runs-on: self-hosted
```

Comment:
changing to use label instead of generic self-hosted will be easier for maintenance

Reply:
yes, we have an issue for that already. But will do a PR changing all those at once.